docs/index.rst

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
a space required at the top of the file, or and RST anchor (e.g. `.. _confluent-kafka-index:`